internal/providers/pluginfw/resources/library/resource_library.go

```diff
@@ -70,19 +70,29 @@ func (r *LibraryResource) Metadata(ctx context.Context, req resource.MetadataReq
 }
 
 func (r *LibraryResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	attrs, blocks := tfschema.ResourceStructToSchemaMap(LibraryExtended{}, func(c tfschema.CustomizableSchema) tfschema.CustomizableSchema {
+		for field, attribute := range c.ToAttributeMap().Attributes {
+			switch attribute.(type) {
+			case tfschema.StringAttributeBuilder:
+				c.AddPlanModifier(stringplanmodifier.RequiresReplace(), field)
+			case tfschema.SingleNestedAttributeBuilder:
+				c.AddPlanModifier(objectplanmodifier.RequiresReplace(), field)
+			}
+		}
+		// for field, block := range c.ToAttributeMap().Blocks {
+		// 	switch block.(type) {
+		// 	case tfschema.ListNestedBlockBuilder:
+		// 		c.AddPlanModifier(listplanmodifier.RequiresReplace(), field)
```

Comment:
Wanted to do this but it seems like there's no such thing as `listplanmodifier`. TODO - figure this out